vip_spec.xsd

```diff
@@ -579,6 +580,30 @@ reference elements for polling location
     </xs:restriction>
   </xs:simpleType>
 
+  <xs:simpleType name="DistrictType">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="city" />
+      <xs:enumeration value="congressional" />
+      <xs:enumeration value="council" />
```

Comment:
Sorry for missing this in my earlier review of the 1622 list. "Council" is insufficient to disambiguate between city council and county council elections, which quite commonly occur together. That means that in an election where there are both city- and county-council contests, and both VIP and unofficial contest data, we would be unable to serve the VIP data even if it includes unique contests.

I'm not aware of any other commonly-coöccuring pair of district types that would present this problem.